src/query/catalog/src/table.rs

```diff
@@ -162,6 +163,7 @@ pub trait Table: Sync + Send {
         ctx: Arc<dyn TableContext>,
         pipeline: &mut Pipeline,
         need_output: bool,
+        _is_ingest: bool,
```

Comment:
Prefer to add an argument instead the `is_ingest`:
```
pub enum AppendMode {
    // From INSERT operation
    Normal,
    // From COPY operation
    Copy,
}
```

Reply:
done